iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1529,7 +1529,7 @@
 
 "type.boundary.administrative.11" = "Frontera administrativa";
 
-"type.boundary.administrative.2" = "Frontera de país";
+"type.boundary.administrative.2" = "National Border";
```

Comment:
Strange. Did you use the latest master?

Where did we lose those `es-MX` translations?

Reply:
please run `git submodule update --init` inside your OM repo and then re-run `generate_localizations.sh`
(there was a change to the Twine tool submodule a couple of weeks ago, it looks like the change didn't make its way to your repo yet)